swagger1.yaml

```diff
@@ -0,0 +1,382 @@
+- name: Centers
+  description: Centers where Doctors belong or work
+paths:
+  /users:
+    post : 
+        tags :
+        - Users
+        summary: Creating new user
+        operationId: addDoctor
+        consumes :
+        - application/json
+        produces :
+        - appication/json
+        parameters:
+        - in: body
+          name: body
+          description: user object that needs to be added to the store
+          required: true
+          schema:
+           $ref: '#/definitions/User'   
+        responses:
+          200:
+            description: User created successfully
+          400:
+            description: Bad request
+          500:
+            description: Internal Server Error  
+
+    get:
+      tags:
+      - Users
+      summary: List of all user
+      operationId: doctorList
+      consumes:
+      - application/json
+      produces: 
+      - application/json
+      parameters:
+      - in: path
+        name: body
+        description: Getting List of all users
+      responses:
+        500:
+          description: Internal Server Error 
+  
+  /users/{id}:        
+    patch:
+      tags:
+      - Users
+      summary: Edit User
+      operationId: editDoctor
+      consumes: 
+      - applicationm/json
+      produces:
+      - application/json
+      parameters:
+      - in: body 
+        name: body 
+        description: Edit a User
+        required: true
+        schema:
+          $ref:
+          type: object
+          properties:
+            username: 
+              type: String
+              example: admin
+            password:
+              type: String
+              example: 1234568963
+      response:
+        200: 
+          description: User updated
+        404:
+          description: User not found
+        500:
+          description: Internal Server error         
+
+  
+    delete:
+      tags:
+      - Users
+      summary: Delete a Doctor
+      operationId: deleteDoctor
+      consumes:
+      - application/json
+      produces:
+      - application/json
+      parameters:
+      - in: path
+      name: id
+      description: deleteUser
+      required: true
+      response:
+        200:
+          description: User Deleted
+        404:
+          description: User not found
+        500:
+          description: Internal Server error
+
+    
+
+  /session:
+    post:
+      tags:
+      - Users 
+      summary: Aunthenticate User
+      operationId: userLogin
+      consumes:
+      - application/json
+      produces:
+      - application/json
+      parameters:
+      - in: body
+        name: body
+        required: true
+        description: User login
+        schema:
+         $ref: 
+         type: object
+         properties:
+          username: 
+           type: String
+           example: admin
+          password:
+            type: String
+            example: 12345
+      responses:
+        200:
+         description: user logged in
+        400:
```

Comment:
This should be a 403 forbidden. You cannot create a session if the username or passwprd are invalid